apps/browser/qml/pages/components/PopUpMenu.qml

```diff
@@ -28,7 +28,7 @@ Item {
     signal closed
 
     width: Math.max(Theme.paddingLarge * widthRatio, footer.implicitWidth)
-    height: Math.min(parent.height - Theme.paddingLarge * 2, _menuItemHeight)
+    height: Math.min(parent.height - Theme.paddingLarge * 2, _menuItemHeight, container.contentHeight + footer.height)
```

Comment:
Rebase conflict. You should be able to drop the change after.

Reply:
Rebased. done